templates/account/invoices/index.html

```diff
@@ -0,0 +1,88 @@
+{% extends "advantage/base_advantage.html" %}
+
+{% block title %}Invoices{% endblock %}
```

Comment:
Also this would be better as something like "Your Canonical invoices"

Reply:
We have the robots tag now.

The meta was updated too.